src/group_theory/group_action/basic.lean

```diff
@@ -312,6 +314,10 @@ attribute [to_additive add_action.quotient_action] mul_action.quotient_action
 @[to_additive] instance left_quotient_action : quotient_action α H :=
 ⟨λ _ _ _ _, by rwa [smul_eq_mul, smul_eq_mul, mul_inv_rev, mul_assoc, inv_mul_cancel_left]⟩
 
+@[to_additive] instance right_quotient_action : quotient_action H.normalizer.opposite H :=
+⟨λ b c _ _, by rwa [smul_def, smul_def, smul_eq_mul_unop, smul_eq_mul_unop, mul_inv_rev, ←mul_assoc,
+  ←subtype.val_eq_coe, mem_normalizer_iff'.mp b.2, mul_assoc, mul_inv_cancel_left]⟩
```

Comment:
This one looks odd to me - why do you need to turn the `coe` into a val? Does this work?
```suggestion
  mem_normalizer_iff'.mp b.prop, mul_assoc, mul_inv_cancel_left]⟩
```

Reply:
I think there might also be a single lemma that combines `mul_assoc, mul_inv_cancel_left` here

Reply:
Thanks for pointing out `b.prop`! `library_search` can't find the lemma:
`example {G : Type*} [group G] {x y z : G} : x * (x⁻¹ * y * z) = y * z := by library_search`